Project/Assets/Editor/Annotations/AnnotationInteractionEditor.cs

```diff
@@ -14,20 +14,60 @@ public override void OnInspectorGUI()
     {
         annotation = (Annotation)target;
 
-        string[] options = new string[] { "Import Text File", "Write in Inspector" };
-        annotation.selected = EditorGUILayout.Popup("Import Method", annotation.selected, options);
+
+        string[] typeOptions = new string[] { "Summary Annotation", "Area Annotation" };
+        annotation.annotationType= EditorGUILayout.Popup("Annotation Type", annotation.annotationType, typeOptions);
+
+        if(annotation.annotationType == 0)
+        {
+            DisplaySetSummary();
+        }
+        else
+        {
+            ///Stuff for Area Annotations here
+        }
+
+        int before = annotation.importType;
+
+        string[] importOptions = new string[] { "No Full Annotation", "Import Text File", "Write in Inspector" };
+        annotation.importType = EditorGUILayout.Popup("Import Method", annotation.importType, importOptions);
+
+        //checks with user, then resets full annotation information
+        if (before != 0 && annotation.importType == 0)
```

Comment:
I think `before` could be named a little more clearly here. Perhaps `previousType`?